internal/handlers/device.go

```diff
@@ -298,8 +305,11 @@ func (api *API) CreateDevice(c *gin.Context) {
 		}
 		// allocate a child prefix if requested
 		for _, prefix := range request.ChildPrefix {
-			if err := api.ipam.AssignPrefix(ctx, org.ID, prefix); err != nil {
-				return fmt.Errorf("failed to assign child prefix: %w", err)
+			// Skip the prefix assignment if it's an IPv4 or IPv6 default route
+			if !util.IsDefaultIPv4Route(prefix) && !util.IsDefaultIPv6Route(prefix) {
```

Comment:
See later where I suggest a helper wrapper that turns this into a single function call.